packages/checkout/registry/test/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Internal dependencies
+ */
+import {
+	__experimentalRegisterCheckoutFilters,
+	__experimentalApplyCheckoutFilter,
+} from '../';
+
+describe( 'Checkout registry', () => {
+	const filterName = 'loremIpsum';
+
+	test( 'should return default value if there are no filters', () => {
+		const value = 'Hello World';
+		const newValue = __experimentalApplyCheckoutFilter( {
+			filterName,
+			defaultValue: value,
+		} );
+
+		expect( newValue ).toBe( value );
+	} );
```

Comment:
Another test we can add (we need to mock `CURRENT_USER_IS_ADMIN`) is passing in invalid code (that throws an exception) and making sure previous filter still passed.

Reply:
I had a bad time mocking `CURRENT_USER_IS_ADMIN` so it had different values in two different tests. I ended up splitting them in two files: 1d5b636f0c15c94872417deeed916c35cfe70af5. There might better ways of doing this (let me know if you find how!), but couldn't figure it out how after spending some time on this.